package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "run-script-webpack-plugin",
+  "name": "run-script-webpack-plugin-fork",
```

Comment:
Shouldn't rename

Reply:
Woops, its a remain of me, had to publish it urgently for my own need :)